src/renderer/ui/components/generic/PlatformSpecific.js

```diff
@@ -23,7 +23,7 @@ export default class PlatformSpecific extends Component {
     if (process.platform === this.props.platform) {
       if (!this.props.versionRange) return this.props.children;
 
-      if (semver.validRange(this.props.versionRange) && semver.satisfies(osVersion, this.props.versionRange)) {
+      if ((semver.validRange(this.props.versionRange) && semver.satisfies(osVersion, this.props.versionRange)) || (osVersion === null && process.platform === 'linux')) {
```

Comment:
If `osVersion` is `null` we should not render regardless of platform.  Also this appears to short-circuit the wrong way.  The `semver.satisfies` check will called on `null` still.

Reverse the "or" order and don't check the platform 👍 

Trying to keep this components in a state where they could be published separately